### ISSUE TITLE
What do you do with new "Recipes" file

### ISSUE THREAD

**Issue by bobstar-fr** [tomy137/mcc_recipies_convert_toSQL#6]
Hello
When you create or modify a recipe :
Can you tell us how you transfer the modified "Recipes" file (with  "Fausses crèpes") to the MCC.
Do you replace the old MCC "Recipes" file with the new "Recipes" file. 
What do you do with "Recipes-journal". 
You delete the old one before restarting the MCC or you leave the old one as is.
Thank you

**Comment by bobstar-fr** [comment]
Hello
Can you tell us how you transfer your recipe for "fausses crèpes" to the MCC after creating the file "file" with your script.
Thanks, it can help the community.

**Comment by tomy137** [comment]
Hello, the file generated by the script is just a list of SQL command to be passed on the recipe file. The Recipe file is, in fact, a .sqlite file. It can be opened by a software like https://sqlitebrowser.org/ that will allow you to pass command on it. 

So open the recipe file with a software like sqlitebrowser.org and copy paste commands of the file generated by this script. Don't forget to save the modifications and upload the new recipe file to the MCC.

**Comment by bobstar-fr** [comment]
Hello
Ok, I understand how your script works.
I work on your scripts and I saw that I cannot create my own recipe in the state and even with the help of "new_recipe_withcomments_FR.json" because it lacks fields and that I do not have enough experience with this programming.
I modified a recipe by manually inserting the new rows in the tables with the DB browser ... and it works.
**Thank you for your help because we are few to tinker on this subject.**

I studied the architecture of the "Recipes" file and I was able to identify the use of all the fields as MODE "ramp" or "level" and that the most important thing when we want to create our recipe is to know the list and the detailed number of ingredients and the number of steps in the recipe before anything else.
Knowing that the steps can be sequential as for MODE "ramp".
For example: "**mixing 6 seconds / speed 3 to 9**" involves 3 steps :
**1 step to mix 2 seconds / speed 3**
and **1 step to mix 2 seconds / speed 6**
and **1 step to mix 2 seconds / speed 9**
so we have **3 steps for 1 operation on the screen**
I think i can create my own recipe by adding rows in the tables (with DB Browser) and filling in the necessary fields.
